actions/transactions/update.go

```diff
@@ -49,3 +55,26 @@ func (a *Action) update(c *gin.Context) {
 	contract := mappings.MapToTransactionContract(transaction)
 	c.JSON(http.StatusOK, contract)
 }
+
+// update will update a transaction
+// Update transaction godoc
+// @Summary		Update transaction
+// @Description	Update transaction
+// @Tags		New Transactions
+// @Produce		json
+// @Param		UpdateTransaction body UpdateTransaction true " "
+// @Success		200 {object} models.Transaction "Updated transaction"
+// @Failure		400	"Bad request - Error while parsing UpdateTransaction from request body, tx not found or tx is not associated with the xpub"
+// @Failure 	500	"Internal Server Error - Error while updating transaction"
+// @Router		/v1/transactions/{id} [patch]
+// @Security	x-auth-xpub
+func (a *Action) updateTransaction(c *gin.Context) {
+	var requestBody UpdateTransaction
```

Comment:
We should make separated model as we no longer need the id in body, and in swagger it will appear as it's there.

Reply:
Replaced with UpdateTransactionRequest model.